docs/reference/asyncapi-rules.md

```diff
@@ -0,0 +1,311 @@
+# AsyncAPI Rules
+
+Spectral has a built-in "asyncapi" ruleset for the [AsyncAPI Specification](https://www.asyncapi.com/docs/specifications/2.0.0/).
+
+In your ruleset file you can add `extends: "spectral:asyncapi"` and you'll get all of the following rules applied.
+
+## AsyncAPI v2-only
+
+These rules will only apply to AsyncAPI v2 documents.
+
+### asyncapi2-channel-no-empty-parameter
```

Comment:
We only had version numbers in the rules when a rule only applied to a specific version. We'll never support `asyncapi1` and we dont know what 3 will look like, these rules might apply and we'd have to rename them later. I think we'd be better off naming them all `asyncapi-*` now to avoid breaking later.

Reply:
@philsturgeon Fixed